src/utility/util.jl

```diff
@@ -21,6 +21,17 @@ function _elementwise_mult(a::AbstractTensorMap, b::AbstractTensorMap)
     return dst
 end
 
+"""
+Compute S^(pow) for diagonal matrices `S`
+"""
+function sdiag_pow(S::AbstractTensorMap, pow::Real)
```

Comment:
Could this function somehow be merged with the one below?

Reply:
I think it might be reasonable to have `tol = pow > 0 ? zero(real(scalartype(S))) : _other_tol)`